src/rawcan_comm.cpp

```diff
@@ -0,0 +1,49 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <string.h>
+
+#include <net/if.h>
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <sys/ioctl.h>
+
+#include <linux/can.h>
+#include <linux/can/raw.h>
+
+#include <string>
+#include <iostream>
+#include <cstring>
+
+#include "rawcan_comm.hpp"
+
+namespace roboteq { 
+
+    rawcan_comm::rawcan_comm(canid_t roboteq_can_id = 1, std::string ifname = "can0"): _roboteq_can_id(roboteq_can_id) {
```

Comment:
Is this the correct syntax for the header? I also had to include _ifname in private (I'm assuming we forgot to include it last week)

Reply:
![image](https://user-images.githubusercontent.com/56097508/69999604-08934b00-1527-11ea-9985-214a826ba87d.png)

Reply:
I don't think we need ifname after we create the socket.

Reply:
Also what i meant to say was that the default arguments should also be in the header. So instead of `explicit rawcan_comm(canid_t _roboteq_can_id, std::string _ifname);`, it should be         `explicit rawcan_comm(canid_t roboteq_can_id = 0x1, std::string ifname = "can0");` and the cpp should be:
`rawcan_comm::rawcan_comm(canid_t roboteq_can_id, std::string ifname): _roboteq_can_id(roboteq_can_id) {`